x-pack/plugins/cases/public/components/markdown_editor/types.ts

```diff
@@ -5,13 +5,12 @@
  * 2.0.
  */
 
-import { FunctionComponent } from 'react';
-import { Plugin, PluggableList } from 'unified';
+import type { FunctionComponent } from 'react';
+import type { Plugin, PluggableList } from 'unified';
 // Remove after this issue is resolved: https://github.com/elastic/eui/issues/4688
-import { Options as Remark2RehypeOptions } from 'mdast-util-to-hast';
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I was able to remove this line, as type imports don't violate the `import/no-extraneous-dependencies` rule (unless explicitly configured, see [plugin options](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md))